examples/scripts/3d_examples/3d_basic_spm_asymmetrical_cooling_prismatic.py

```diff
@@ -0,0 +1,94 @@
+import pybamm
```

Comment:
minor, but can we update the filename to use "pouch" and not "prismatic"? otherwise looks great!